week-3/Medha_Mathur/wallpaperChanger.py

```diff
@@ -0,0 +1,25 @@
+import struct
+import ctypes
+
+
+SPI_SETDESKWALLPAPER = 20
+WALLPAPER_PATH = 'C:\\Users\LENOVO\Desktop\weekly-challenges\week-3\Medha_Mathur\wallpaper.jpg'
```

Comment:
can you perhaps add a so-called wallpaper folder, in which you store wallpapers then it cycles through those wallpapers because you are using only one image here so it will work only once and that's not the point.

Reply:
Ok...Let me try.